src/ngraph/util.cpp

```diff
@@ -365,4 +365,14 @@ namespace ngraph
         }
         return result;
     }
+
+    std::vector<std::string> node_vector_to_string(const NodeVector& v)
```

Comment:
`node_vector_to_string_vector`?
I think there's an obscurely named STL function with about a dozen iterator arguments that can do this in one line.
Why not just have a conversion operator on `NodeVector`?

Reply:
yes, there's `transform`  but it will actually be less readable than this and it's indeed more "obscure". Alternatively, I could overload `<<` for `NodeVector` but I don't know if we really need it.